src/middleware/auth.js

```diff
@@ -41,6 +41,16 @@ export async function requiereAuth(req, res, next) {
   }
 }
 
+export function invalidarToken(req, res, next) {
+  try {
+    const token = req.header("Authorization");
```

Comment:
```suggestion
    const token = req.headers.authorization.split(' ')[1];
```
Formato token en header:
```js
// req.headers
{
  ...
  authorization: 'Bearer header.payload.signature',
  ...
}
```

Reply:
@lfazocar Una consulta, autenticacion.routes.js las rutas habría que modificar algo por lo del fix al middleware?

Reply:
no, no es necesario
el problema estaba en /routes/index.js